### PR TITLE
meetup-2023-10-29: shorten text

### PR DESCRIPTION
**Pull request by msanft**
Shorten the text to be more concise

### CODE REVIEW

File: meetups/2023-10-19/announcement.md [hunk 1]
```diff
@@ -1,22 +1,8 @@
...
-
-Ruhr-Universität Bochum, Raum MC 1.30 / MC 1.31. Eine Anfahrtsbeschreibung findet ihr auf der [Webseite der Uni](https://www.ruhr-uni-bochum.de/de/anreise-zur-ruhr-universitaet). Das Gebäude MC ist grün im beigefügten Campusplan eingezeichnet. Die Räume MC 1.30 / MC 1.31 finden sich sehr zentral im ersten Stock. Kontaktiert uns gerne für einen detaillierten Lageplan zur rollstuhlgerechten Anreise.
-
-### Wer?
-
-Absolut jede*r mit Interesse an Nix(OS). Ihr müsst keine Vorerfahrung mit Nix mitbringen.
-
-## FAQ
+- **Wann?** 19.10.2023, ca. 18:00-22:00
+- **Wo?** RUB, Raum MC 1.30 / MC 1.31. Eine Anfahrtsbeschreibung findet ihr auf der [Webseite der Uni](https://www.ruhr-uni-bochum.de/de/anreise-zur-ruhr-universitaet). Die Räume MC 1.30 / MC 1.31 finden sich sehr zentral im ersten Stock. Kontaktiert uns gerne für einen detaillierten Lageplan zur rollstuhlgerechten Anreise.
```
Comment (katexochen) [inline meetups/2023-10-19/announcement.md:4]: > Kontaktiert uns gerne für einen detaillierten Lageplan zur rollstuhlgerechten Anreise.

How? There is no contact information.

**Review by katexochen — APPROVED** [review]
Otherwise lgtm, thanks.

File: meetups/2023-10-19/announcement.md (hunk 1 above)
Comment (msanft) [reply meetups/2023-10-19/announcement.md:4]: There is if we post it in a Discord, etc.

Comment (katexochen) [reply meetups/2023-10-19/announcement.md:4]: I think the ITS folks will post it as event and on the mailing list, so there won't.

Comment (msanft) [reply meetups/2023-10-19/announcement.md:4]: Newton will forward to me / us, both per mail and Discord

**Pull request merged**